pkg/dyninst/gosym/symtab.go

```diff
@@ -1180,3 +1193,86 @@ func stringFromOffsetPtr(data []byte, offset int) *string {
 	result := string(data[offset:end])
 	return &result
 }
+
+// FuncPCResolver resolves program counters within a single function to source
+// line numbers. It is optimized for resolving multiple PCs in increasing order.
+type FuncPCResolver struct {
+	funcPC uint64
+	lt     *lineTable
+
+	stepper *stepper
```

Comment:
nit: no need to have this be a pointer, just make it part of the struct and change `newStepper` to `makeStepper`?